### ISSUE TITLE
Replace google groups link with one for virus browser

### ISSUE THREAD

**Issue by jaflynn5** [debugpoint136/virus-browser#22]
In the "Help" drop-down menu, the link "Google groups" points to a question and answer page for the Epigenome Browser. Since most of the comments there are irrelevant to the Virus Browser, can we make a new page for the Virus Browser and have that link replace the current link?

**Comment by debugpoint136** [comment]
Please confirm if still an issue @jaflynn5

**Comment by jaflynn5** [comment]
Yes, the "Help" menu no longer displays. Should we still create a "Google Groups" for the virus browser though? We could provide a link under the "Resources" tab.
